python/tskit/combinatorics.py

```diff
@@ -255,7 +255,7 @@ def join_topologies(child_topologies):
         return RankTree(children).rank()
 
 
-def all_trees(num_leaves):
+def all_trees(num_leaves, **kwargs):
```

Comment:
I'm not sure about this - why not just have ``span=None`` here? If we don't expect to have lots of parameters, then it's just making it hard for users to know where parameters come from. It's the same amount of typing, for now.

Reply:
I um-ed and ah-ed about putting it in. The argument for doing so is that we might want to add other features such as specifying branch length or node labels, which we would simply add to the underlying `to_tsk_tree()` method, and then get all the rest for free. But I could explicitly specify `span` for the time being (and repeat the docstring description for all the methods that call  `to_tsk_tree()`)